src/snyk/common/commands/commandController.ts

```diff
@@ -52,6 +53,10 @@ export class CommandController {
     return this.executeCommand(SNYK_LOGIN_COMMAND, this.authService.initiateLogin.bind(this.authService, getIpFamily));
   }
 
+  initiateLogout(): unknown {
```

Comment:
Any reason for unknown return type here?

Reply:
Because the underlying `executeCommand` function returns a Promise of `unknown` type which further calls Lodash's `debounce` function which returns an `unknown` type. 😅 

I guess in this case we are just executing the command without really needing to know the type since we are not consumsing the type anywhere else.